### PR TITLE
[gui] fix some fallback icons in Art selection dialog

### PR DESCRIPTION
**Pull request by phil65**
we currently use a wrong fallback icon for the "no art" entry in art selection dialog.
DefaultVideo.png is normally used for Video files and similar, DefaultPicture.png is more appropriate in this case.

@ronie @BigNoid 

Perhaps I should also add a defaulticon to the thumb://Current entry?

EDIT: added fallback icons for all entries which had none.

### CODE REVIEW

**Comment by BigNoid** [comment]
Yeah makes sense to have default picture icon there. Not sure if it's worth the trouble for the thumb://current entry as that is hidden when there's no art and in cases where it's empty it's because the url image can't be loaded, but does has a path to the image. In which case the icon never kicks in.

**Comment by phil65** [comment]
yeah but skinners can also use .ActualIcon and thus explicitely use the icon image.

**Comment by BigNoid** [comment]
That's kind of an edge case for that dialog I think, but sure no objections if you want to add it.

**Comment by ronie** [comment]
no objections from me either

**Comment by phil65** [comment]
jenkins build this please

File: xbmc/video/dialogs/GUIDialogVideoInfo.cpp [hunk 1]
```diff
@@ -794,6 +797,7 @@ void CGUIDialogVideoInfo::OnGetFanart()
   {
     CFileItemPtr itemCurrent(new CFileItem("fanart://Current",false));
     itemCurrent->SetArt("thumb", m_movieItem->GetArt("fanart"));
+    itemNone->SetIconImage("DefaultPicture.png");
```
Comment (Rechi) [inline xbmc/video/dialogs/GUIDialogVideoInfo.cpp]: maybe `itemCurrent` instead of `itemNone`

File: xbmc/video/dialogs/GUIDialogVideoInfo.cpp [hunk 2]
```diff
@@ -819,6 +823,7 @@ void CGUIDialogVideoInfo::OnGetFanart()
   {
     CFileItemPtr itemLocal(new CFileItem("fanart://Local",false));
     itemLocal->SetArt("thumb", strLocal);
+    itemNone->SetIconImage("DefaultPicture.png");
```
Comment (Rechi) [inline xbmc/video/dialogs/GUIDialogVideoInfo.cpp]: maybe `itemLocal` instead of `itemNone`

File: xbmc/video/dialogs/GUIDialogVideoInfo.cpp (hunk 1 above)
Comment (phil65) [inline xbmc/video/dialogs/GUIDialogVideoInfo.cpp]: oops sorry, c&p ftw :/

**Comment by phil65** [comment]
jenkins build this please

**Pull request merged**